ui/panels-plugin/src/plugins/line-chart/tooltip/Tooltip.tsx

```diff
@@ -28,14 +28,17 @@ function assembleTransform(coords: Coordinate, chartWidth: number) {
 
 interface TooltipProps {
   tooltipData: TooltipData;
+  /**
+   * Specify if the labels should wrap instead of truncate.
+   */
+  wrapLabels?: boolean;
```

Comment:
I was debating whether or not to include label wrapping in this PR, so if out of scope or if we want to take another approach I am open to removing this for now.

Reply:
Sounds good to be a separate PR / something we iterate on later. 

I am pleasantly surprised with how the wrapped labels look in Figma. My opinion would be to reduce TOOLTIP_MAX_WIDTH some and allow labels to wrap to two lines (then in the rare case that it needs to wrap to three lines, they get truncated). But definitely go with whatever you think is best, this looks way better than what I came up with!

Reply:
yeah I think it will be interesting to do it in another PR. Like that it will be easy to see the before/after the changes